src/Contexts/ScorecardInfoProvider.js

```diff
@@ -0,0 +1,33 @@
+import React, { createContext, useState } from 'react'
+
+export const ScorecardInfo = createContext()
+
+export const ScorecardInfoProvider = ({ children }) => {
+
+  const [ scorecard, setScorecard ] = useState({
```

Comment:
We should consider splitting this up in the future. This might be too much to keep track of in one spot.